wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/ConversationMessageCell.swift

```diff
@@ -143,6 +146,10 @@ protocol ConversationMessageCellDescription: AnyObject {
     /// The view that will be displayed for the cell.
     associatedtype View: ConversationMessageCell, UIView
 
+    typealias ZMConversationMessage = WireDataModel.ZMConversationMessage
+
+    var conversationCellModel: ConversationCellModel? { get }
```

Comment:
I added the new model to the cell description in order to allow incremental adoption of the new approach.
On the long run the model should replace the cell description.

Reply:
suggestion: add this comment to the source code.